core/src/main/java/org/apache/carbondata/core/datastore/impl/FileFactory.java

```diff
@@ -606,4 +609,53 @@ public static FileSystem getFileSystem(Path path) throws IOException {
     return path.getFileSystem(configuration);
   }
 
+  // Get the total size of carbon data and the total size of carbon index
+  public static HashMap<String, Long> getDataSizeAndIndexSize(CarbonTablePath carbonTablePath,
```

Comment:
This method cannot be in FileFactory, as it in not generic filesystem operation.

Reply:
handled comments